src/Badge.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import { space } from 'styled-system'
+import { theme as themeGet, space } from 'styled-system'
```

Comment:
Saw you had to use alias for the imported theme as `themeGet` and the theme utility all works. +1 Really

Non-blocking, tiny nit on the name of the alias, if you agree that looking at the alias signature, it seems to be in a form of invokable signature with  "**_nounVerb_**" (i.e. "somethingDo"), I wonder if the signature can simply swap the order of the words to follow like the traditional CS naming invokable signature to be something closer to "_**verbNoun**_" (i.e. `doSomething`)?
```
themeGet('letterSpacings.caps')
```